src/app/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap

```diff
@@ -143,6 +143,7 @@ exports[`PodcastExternalLinks Should render external links 1`] = `
         dir="ltr"
       >
         <a
+          aria-label="Apple, A brand podcast , внешняя ссылка"
```

Comment:
Not sure we should have an extra space character after 'A  brand podcast' and the comma afterwards?

Reply:
Also unsure why we have mix languages here? e.g. some english and some russian text?

Reply:
Should the example url be updated to be external?

Reply:
The comma after 'A brand podcast' is taken from the externalLinkText in the config file, but the space before it can be removed. I believe the English text and URL are just placeholders for testing purposes only.

Reply:
Thanks, while they are placeholders, it's always nice to have content that is realistic and reflects the test :8